src/Content.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component, Children } from 'react'; // eslint-disable-line no-unused-vars
+import PropTypes from 'prop-types';
+
+export default class Content extends Component {
```

Comment:
Could you add a test for this component too please

Reply:
@ryanseddon done